apps/citizen_advocate/web/models/user.ex

```diff
@@ -65,4 +68,42 @@ defmodule CitizenAdvocate.User do
     end
   end
 
+  defp downcase_email(changeset) do
+    email = get_change(changeset, :email)
+    if is_nil(email) do
+      changeset
+    else
+      put_change(changeset, :email, String.downcase(email))
+    end
+  end
+
+  def confirmation_changeset(user, params = %{"token" => token}) do
+    user
+    |> cast(params, [])
+    |> put_change(:hashed_confirmation_token, nil)
+    |> put_change(:confirmed_at, Ecto.DateTime.utc())
+    |> validate_token(token)
+  end
+
+  alias Comeonin.Bcrypt
```

Comment:
According to the docs:
>import/2, require/2 and alias/2 are called directives and all have lexical scope. This means you can set up aliases inside specific functions and it won’t affect the overall scope.

Not sure if that means it's hoisted to the top of the declared scope, but if you want to move this into the `validate_token` function I think that's fine. Otherwise I'd move it to the top (only because I tend to look at the top for import/alias/require/use declarations)